src/Tizen.NUI/src/internal/Interop/Interop.TextEditor.cs

```diff
@@ -239,6 +239,12 @@ internal static partial class TextEditor
             //manual pinvoke for text-editor ScrollStateChangedSignal
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_TextEditor_ScrollStateChangedSignal")]
             public static extern global::System.IntPtr TextEditor_ScrollStateChangedSignal(global::System.Runtime.InteropServices.HandleRef jarg1);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_TextEditor_Property_SELECTED_TEXT_START_get")]
+            public static extern int TextEditor_Property_SELECTED_TEXT_START_get();
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_TextEditor_Property_SELECTED_TEXT_END_get")]
+            public static extern int TextEditor_Property_SELECTED_TEXT_END_get();
```

Comment:
Please check the api name used in C # binder.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/244453/